task-sdk/tests/task_sdk/execution_time/test_task_runner.py

```diff
@@ -974,7 +974,7 @@ def test_get_context_with_ti_context_from_server(self, create_runtime_ti, mock_s
             "ti": runtime_ti,
             "dag_run": dr,
             "data_interval_end": timezone.datetime(2024, 12, 1, 1, 0, 0),
-            "data_interval_start": timezone.datetime(2024, 12, 1, 0, 0, 0),
+            "data_interval_start": timezone.datetime(2024, 12, 1, 1, 0, 0),
```

Comment:
Hmm why?

Reply:
This is because of the changes to `create_runtime_ti` fixture. Previously it had hard-coded values. 

https://github.com/apache/airflow/blob/965a1cd7f25867ce7430397d58249a0d05e77ebb/task-sdk/tests/conftest.py#L362-L363

I changed that to dynamically generate those values:

https://github.com/apache/airflow/blob/23f5210290579504a79296a95e383f7967953838/task-sdk/tests/conftest.py#L405-L411


This was part of my adding a new `run_task` fixture:

https://github.com/apache/airflow/blob/23f5210290579504a79296a95e383f7967953838/task-sdk/tests/conftest.py#L459-L475